R/Widget_TimeSeries.R

```diff
@@ -97,11 +98,16 @@ Widget_TimeSeries <- function(
 
   }
 
+  if (!is.null(dfSite)) {
+    dfSite <- jsonlite::toJSON(dfSite, na = "string")
+  }
+
   # forward options using x
   x <- list(
     dfSummary = jsonlite::toJSON(dfSummary, na = "string"),
     lLabels = lLabels,
     dfParams = dfParams,
+    dfSite = dfSite,
```

Comment:
@samussiah Assuming new version of rbmviz knows how to handle NULL for dfSite?

Reply:
@jwildfire - yep, completeely optional.